<!--
 Copyright 2022 Shadow Robot Company Ltd.
 This program is free software: you can redistribute it and/or modify it
 under the terms of the GNU General Public License as published by the Free
 Software Foundation version 2 of the License.
 This program is distributed in the hope that it will be useful, but WITHOUT
 ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
 FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
 more details.
 You should have received a copy of the GNU General Public License along
 with this program. If not, see <http://www.gnu.org/licenses/>.
-->

<launch>
  <arg name="mapping" default="$(find sr_cyberglove_config)/mappings/GloveToHandMappings_generic"/>

  <!-- shadowhand to cybergrasp remapper -->
  <node pkg="sr_remappers" name="cyberglove_remapper"
	type="cyberglove_remapper">
    <param name="cyberglove_prefix" type="string"
    value="/cyberglove/" />
    <param name="sendupdate_prefix" type="string"
    value="/srh/" />
    <param name="cyberglove_mapping_path" type="string" value="$(arg mapping)" />
  </node>
</launch>

